### DISCUSSION TITLE
Will there be more pinned location?

### DISCUSSION THREAD

**Discussion by sapphired18** [rocksdanister/weather#22] [Q&A]
the current implementation is only limits to 3, can it be more?

**Comment by sapphired18** [comment]
I know it's a bit silly to have 10 different location to monitors

**Reply by rocksdanister** [reply]
I restricted to 3 to reduce API calls for the provider:

https://github.com/rocksdanister/weather/blob/af51632fe16fe9e9e00455b3f7c458da0593590b/src/Drizzle.UI.Shared/ViewModels/ShellViewModel.cs#L84

https://github.com/rocksdanister/weather/blob/af51632fe16fe9e9e00455b3f7c458da0593590b/src/Drizzle.Common/Constants/UserSettingsConstants.cs#L42

What is the use case for 10 pins 😲?
